packages/editor/src/components/table-of-contents/panel.js

```diff
@@ -20,9 +21,16 @@ function TableOfContentsPanel( { headingCount, paragraphCount, numberOfBlocks, h
 				aria-label={ __( 'Document Statistics' ) }
 				tabIndex="0"
 			>
+				{
+					__( 'Words' ) !== __( 'Characters' ) &&
```

Comment:
What is the goal of this? Do you rely on translators on changing this? That's not possible as there is no context.

Reply:
@swissspidy Ah, yes, I meant https://github.com/WordPress/gutenberg/pull/14589#issuecomment-490989857

Reply:
We should export the translation in both count components, then compare the types here. https://github.com/WordPress/gutenberg/pull/14589/files#r282803952